Ginger/GingerCoreNET/Database/NoSqlBase/GingerHbase.cs

```diff
@@ -0,0 +1,611 @@
+                                string[] selectedcols = SQLCalculated.Trim().Substring((SQLCalculated.IndexOf("select") + 6), SQLCalculated.IndexOf("from") - SQLCalculated.IndexOf("select") -6).Split(",");
+                                CellSet next;
+                                List<string> list = new();
+                                foreach (string col in selectedcols)
+                                    list.Add(col.Trim());
+                                
+                                while ((next = client.ScannerGetNextAsync(scanInfo, scanOptions).Result) != null)
+                                {
+                                    foreach (CellSet.Row row in next.rows)
+                                    {
+                                        string rowKey = _encoding.GetString(row.key);
+
+                                        List<Cell> cells = row.values;
+
+                                        foreach (Cell c in cells)
+                                        {
+                                            string colname = ExtractColumnName(c.column);
+                                            if (list.Contains(colname))
+                                            {                                                
+                                                Act.AddOrUpdateReturnParamActualWithPath(colname, Encoding.ASCII.GetString(c.data), path.ToString());                                               
+                                               
+                                            }
+                                            
+                                        }
+                                        path++;
+                                    }
+                                }
+
+                            }
+                        }
+                       
+                    }
+                    finally
+                    {
+                        if (scanInfo != null)
+                        {
+                            client.DeleteScannerAsync(table, scanInfo, scanOptions).Wait();
+                        }
+                    }
+
+                }
+
+            }
+            catch (Exception ex)
+            {
+                Act.Error = ex.Message;
+                Reporter.ToLog(eLogLevel.ERROR, ex.Message, ex);
+            }
+        }
+
+
+        public string Base64Encode(string text)
+        {
+            var textBytes = System.Text.Encoding.UTF8.GetBytes(text);
+            return System.Convert.ToBase64String(textBytes);
+        }
+        public string Base64Decode(string base64)
+        {
+            var base64Bytes = System.Convert.FromBase64String(base64);
+            return System.Text.Encoding.UTF8.GetString(base64Bytes);
+        }
+        public override List<string> GetKeyspaceList()
+        {
+            throw new NotImplementedException();
+        }
+
+        public override List<string> GetTableList(string Keyspace)
```

Comment:
We have code to get table list above in the class. please add that code in GetTableList function and use this function to get table list